docs/changes/coordinates/15220.api.rst

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Nitpick: no need for an empty line

Reply:
That's from pre-commit's end-of-file fixer. All files end with an empty line (except those excluded from the CI check).

Reply:
Done.